### DISCUSSION TITLE
Issues tab

### DISCUSSION THREAD

**Discussion by russelporosky** [alpinejs/alpine#3920] [4. General]
The issues tab is often used as a very visible indicator of activity for a repository. Discussions is much more opaque.

For example, with Issues, I can click on Bug and see immediately how many open and closed bug topics there are. With discussions, I need to count the number of posts on a page and multiply that by the number of pages.

The Issues tab included a badge showing the number of open issues, whereas discussions doesn't.

I believe (and this is how I run my repos) that Issues is for actual issues and features. Discussions is where discussion about those things happen. So if there's a bug, I'd expect to see an Issue for it, and optionally a Discussion arguing the merits of a specific way of dealing with that bug and it's potential consequences.

I would suggest that if you felt that Issues was being abused, then Discussions should have been the extension and not the replacement. It's less useful from a project standing perspective. Issues is for changes to the code, Discussions is for conversations around those changes (and the project itself, natch).

**Comment by ekwoka** [comment]
While that does make some sense there is alternative discussion regarding this: https://github.com/orgs/community/discussions/5709

Broadly, 99% of the current discussions are PEBKAC stuff, the majority being solved by RTFM.

There is a benefit that these can be addressed by the community and not require a maintainer to close, etc

It's a balance of various factors, and I don't think that either issues > discussion or discussions > issues makes perfect sense for every project that get considerable use.

**Comment by russelporosky** [comment]
It seems that "Convert to Discussion" and "Convert to Issue" buttons might be handy. Issues are for issues, discussions are for discussing. Maybe Issues should just be Bugs (narrow the focus a bit) and everything else is a Discussion (eg. Feature Requests could generally use some good conversation before planning in many projects' cases)?

**Comment by ekwoka** [comment]
I think also GitHub providing some more granular accesses could be good.

I know there is some different levels, but I'm not sure how much they allow for people to not have any actual write access but still have "community" access.

Like being able to have active volunteers promote and demote discussion/issues.

And maybe some more granular options for indicating an issue as closed because it's not an actual issue (or actually an issue with another project, especially with Alpine and Livewire).

Plenty that could be done.

I do generally agree that at least on the case of Alpine, issues themselves aren't necessarily an important thing.

Idk I'd you've listened to Caleb's podcast episodes on the topic, but it can at least shed light on why this is the situation with Alpine itself, regardless of what may be best for other orgs and projects.
